contracts/LivepeerProtocol.sol

```diff
@@ -121,6 +121,22 @@ contract LivepeerProtocol {
     // rewardMultiplier[1] -> transcoder's cumulative stake for round
     mapping (address => mapping (uint256 => uint256[2])) public rewardMultiplierPerTranscoderAndRound;
 
+    // The various states a job can be in
+    enum JobStatus { Inactive, Unclaimed, Claimed, Verified }
```

Comment:
The values for this enum are the states that a job can be in. A job is `Unclaimed` after a broadcaster submits a transcoding job by calling `job()`. A job is `Claimed` after a transcoder calls `endJob()` and submits on-chain the merkle root of the transcode claims for a range of segments. A job is `Verified` after a transcoder successfully calls `verify()` for all segments that are eligible for verification (determined after `endJob()`).

Not sure if we need an `Inactive` state. We might use it if we introduce a function to "close" a job if a transcoder is not being responsive or if a broadcaster does not want to live stream after a period of time during which no transcoder claims work for the job. Or we can just leave jobs that fall under these circumstances as `Unclaimed` .